src/config.js

```diff
@@ -0,0 +1,29 @@
+window.AFRAME_CONFIG = window.AFRAME_CONFIG || {};
+
+var isIOSOlderThan10 = require('./utils/isIOSOlderThan10');
+// Workaround for iOS Safari canvas sizing issues in stereo (webvr-polyfill/issues/102).
+// Only for iOS on versions older than 10.
+var bufferScale = isIOSOlderThan10(window.navigator.userAgent)
+  ? 1 / window.devicePixelRatio
+  : 1;
+
+var DEFAULT_CONFIG = {
+  ASSET_BASE_URL: 'https://cdn.aframe.io/',
+  polyfillConfig: {
```

Comment:
I prefer to check internally if it's cordova instead of increasing public API surface that we would have to document, maintain and be sensible when changed.

Reply:
Agreed, a few questions then:

1. How can I change the hard-coded `cdn.aframe.io` references to point to local resources instead?
2. How can I point polyfillConfig's `DPDB_URL` to a local resource?